tests/test_pipeline/test_performance.py

```
@@ -0,0 +1,38 @@
+CONFIG_PATH = "./corebehrt/configs/evaluate_finetune.yaml"
+
+
+def main_evaluate_performance(config_path):
+    # Setup directories
+    cfg = load_config(config_path)
+    bad_on_bad_metrics_path = cfg.paths.bad_on_bad_metrics_path
+    bad_on_good_metrics_path = cfg.paths.bad_on_good_metrics_path
+    good_metrics_path = cfg.paths.good_metrics_path
+
+    bad_on_bad_metrics = pd.read_csv(bad_on_bad_metrics_path)
+    bad_on_good_metrics = pd.read_csv(bad_on_good_metrics_path)
```

Ruff (0.11.9)

Local variable `bad_on_good_metrics` is assigned to but never used

Remove assignment to unused variable `bad_on_good_metrics`

(F841)